bin/moodle-docker-compose

```diff
@@ -128,6 +128,11 @@ then
     dockercompose="${dockercompose} -f ${basedir}/phpunit-external-services.yml"
 fi
 
+if [[ ! -z "$MOODLE_DOCKER_BEHAT_FAILDUMP" ]];
```

Comment:
Hi @aspark21,
what about testing even if it is actually a directory too?
Kind of `[[ -d "$MOODLE_DOCKER_BEHAT_FAILDUMP" ]]`.

HTH,
Matteo

Reply:
Fair point updated accordingly